hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/impl/HddsDispatcher.java

```diff
@@ -261,6 +261,7 @@ private ContainerCommandResponseProto dispatchRequest(
         // If container does not exist, create one for WriteChunk and
         // PutSmallFile request
         responseProto = createContainer(msg);
+        metrics.incContainerOpsLatencies(Type.CreateContainer);
```

Comment:
Latency needs time as second parameter:

https://github.com/apache/ozone/blob/334c6ee65bdc41d0150ad381bbed024a82ea0081/hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/impl/HddsDispatcher.java#L315-L316

Reply:
oh! I found I actually just need to call incContainerOpsMetrics right before incContainerOpsLatencies at line 315. I just moved it!

Reply:
update: we shall not move this line to 315, we keep this here. And I added the startTime to calculate the latency.